Library/Homebrew/cask/lib/hbc/utils/quarantine.swift

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/swift
+
+import Foundation
+import CoreServices
+
+let QUARANTINE_SUCCESS: Int = 0
+let QUARANTINE_FAILURE: Int = 1
```

Comment:
With `0/1`, I meant just using `exit(1)` instead of `exit(Int32(QUARANTINE_FAILURE))`.